docs/overview.md

```diff
@@ -6,7 +6,7 @@ This page describes checks supported by [go-critic](https://github.com/go-critic
 
 ## Checkers
 
-Total number of checks is 101 :rocket:
+Total number of checks is 65 :rocket:
```

Comment:
hm...

Reply:
Oh. I think we need to register embedded rules explicitly, from the main of the docs generator.